packages/node_modules/@webex/webex-core/src/interceptors/auth.js

```diff
@@ -103,7 +103,7 @@ export default class AuthInterceptor extends Interceptor {
     return services.waitForService({name: service, url: uri})
       .then((detectedUrl) => {
         // Validate if the url was detected while waiting for the service.
```

Comment:
This comment needs to be updated to match what we are doing here

Reply:
fixed this.